view/desktop/src/App.tsx

```diff
@@ -69,7 +63,7 @@ const App: React.FC = () => {
           </ResizablePanel>
           <ResizablePanel>
             <ContentLayout className="content relative flex h-full flex-col overflow-auto">
-              <Suspense fallback={<div className="loading">Loading...</div>}>
+              <Suspense fallback={<div>Loading...</div>}>
```

Comment:
Why are we still using this? Can we use our loading screen component here instead?

Reply:
Because this loader is inside the contet layout. Since this loader is nested within the content layout, placing the loading screen component here would briefly display a fullscreen loader, followed by this component for a split second. It kind of looks like a bug if you don't know that it is supposed to be there.
![изображение](https://github.com/user-attachments/assets/ea5fbe77-5a84-48f4-b455-bd93760a0caf)